libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/route/RouteFetcher.java

```diff
@@ -70,18 +72,15 @@ public void clearListeners() {
    * @since 0.13.0
    */
   public void findRouteFromRouteProgress(Location location, RouteProgress routeProgress) {
-    if (isInvalidProgress(location, routeProgress)) {
-      return;
-    }
     this.routeProgress = routeProgress;
-    NavigationRoute.Builder builder = buildRequestFromLocation(location, routeProgress);
-    executeRouteCall(builder);
+    NavigationRoute.Builder builder = buildRequest(location, routeProgress);
+    findRouteWith(builder);
   }
 
   @Nullable
-  private NavigationRoute.Builder buildRequestFromLocation(Location location, RouteProgress progress) {
+  public NavigationRoute.Builder buildRequest(Location location, RouteProgress progress) {
```

Comment:
Why was this renamed and why is this public now?

Reply:
Same as above i adapted this to the new mapbox version